docs/topics/process.rst

```diff
@@ -71,14 +71,14 @@ The example below shows an orders-reservations-payments system.
         class Created(Event, AggregateRoot.Created):
             pass
 
-        def reserved(self):
+        def reserve(self):
```

Comment:
Thanks @vladimirnani, I agree this method could be renamed, but I'm not sure that's the right place for that verb. This method could perhaps "usefully" put the new reservation ID into the order object, and be named accordingly, somehow? Similarly for the code that responds to the payment being created, the payment aggregate ID could be added to the order aggregate? The example demonstrates that one process can follow two other processes, and hence centralised the coordination of a distributed multi-aggregate, multi-step sequence. But it's a terrible orders-reservations-payments system, so it would be nice to make it a more realistic. For example, what gets reserved? and what's the behaviour when there aren't any left? how does a user provide card details, or at least confirm their payment? How would a third-party payment provider be integrated into this system? And is there a view that collects all the orders for a user? Do they perhaps get tickets, so they can claim what they have reserved and paid for? We could probably go mad and make it insanely complicated, but before that, there's probably space in that doc to make a slightly more realistic system? We could kick it into shape I'm sure, it's basically common sense. But I wondered, is there maybe a conventional way of doing it, that people would recognise?